iFixit/pulldasher
# Webpack: Bundle JS with webpack

This replaces require.js and bower with webpack and npm for bundling and loading front end JavaScript files. By using webpack, this makes it easy for us to make use of loaders to incorporate things like babel, uglify, eslint, and maybe even typescript into our build process.

This pull does these main things:
* All uses of `define(['jquery', 'underscore'], function($, _) {`, etc. are replaced with ES6 import statements.
* All modules handled by bower are now handled by npm

### CR Notes

A huge amount of the diff is just whitespace/tabs. `?w=1` is your friend.

### QA Notes

~~This should work the same as `master` does. I'd recommend going with the docker-compose route to make sure that you can spin up a local dev environment and have it work off the bat.~~

This has been sitting/being used in staging for over a day and nothing crazy has happened, so I'm inclined to just merge it.

qa_req 0

Closes #97
